wazo_dird/plugins/office365_backend/schemas.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+from xivo.mallow_helpers import ListSchema as _ListSchema
+from wazo_dird.schemas import BaseSourceSchema
+from xivo.mallow import fields
+
+from xivo.mallow.validate import Length
+
+
+class SourceSchema(BaseSourceSchema):
+
+    auth = fields.Dict(
+        keys=fields.String(validate=Length(min=1, max=64)),
+        values=fields.String(validate=Length(min=1, max=64))
+    )
+    endpoint = fields.String(
+        missing='https://graph.microsoft.com/v1.0/me/contacts',
+        validate=Length(min=1, max=255),
```

Comment:
Does Microsoft provides this service on premise or we have something only for testing in our API ?

Reply:
I don't know if Microsoft offers this service on premise. I think the parameter was there when the proof of concept was written, then it got into the tests and made it to production.

I would not mind replacing the argument with a config option to make the API simpler in the future.